mixins/demo/labelled-mixin.html

```diff
@@ -21,6 +21,7 @@
 					`;
 				}
 				render() {
+					if (!this.validateLabel()) return;
```

Comment:
This will be the change that controls will need to opt into to get label validation. For new controls or ones where we're certain labels are always provided, they can `return` right away and not render if the label is missing. For existing ones though we'll want to just validate but continue rendering until we're sure we've caught them all.